xbmc/addons/PVRClient.h

```diff
@@ -681,6 +708,13 @@ namespace PVR
     static void WriteClientChannelInfo(const CPVRChannelPtr &xbmcChannel, PVR_CHANNEL &addonChannel);
 
     /*!
+     * @brief Copy over epg info from CEpgInfoTag to EPG_TAG.
+     * @param tag The epg tag on Kodi's side.
+     * @param pvrTag The epg tag on the addon's side.
+     */
+    static void WriteEpgTag(const EPG::CConstEpgInfoTagPtr &tag, EPG_TAG &pvrTag);
```

Comment:
if you use EPG_TAG.iUniqueBroadcastId in the API you do not need this (expensive) method.